index.test-d.ts

```diff
@@ -0,0 +1,20 @@
+import {expectType} from 'tsd-check';
+import {default as callsites, CallSite} from '.';
+
+const callsite = callsites()[0];
+
+expectType<CallSite[]>(callsites());
+
+expectType<unknown | undefined>(callsite.getThis());
+expectType<string | null>(callsite.getTypeName());
+expectType<Function | null>(callsite.getFunction());
+expectType<string | null>(callsite.getFunctionName());
+expectType<string | undefined>(callsite.getMethodName());
+expectType<string | null>(callsite.getFileName());
+expectType<number | null>(callsite.getLineNumber());
+expectType<number | null>(callsite.getColumnNumber());
+expectType<string | undefined>(callsite.getEvalOrigin());
+expectType<boolean>(callsite.isToplevel());
+expectType<boolean>(callsite.isEval());
+expectType<boolean>(callsite.isNative());
+expectType<boolean>(callsite.isConstructor());
```

Comment:
Do you really see the value of testing all of the methods? They can't really be anything other than what we define them to be, so this test is not really testing anything.

Reply:
Was wondering the same. The only thing it could help with is if someone removes one of them out of the `d.ts` file. Don't really have a strong opinion on it.